### PR TITLE
Use latest rust tooling for Gitpod

### PR DESCRIPTION
**Pull request by haraldreingruber**
This PR uses the latest rust docker image, as a base for the dev environment on Gitpod.

Use this URL to test the changes: https://gitpod.io/#https://github.com/mob-programming-meetup/advent-of-the-rustic-mob/tree/gitpod-rust-base-image

### CODE REVIEW

**Comment by haraldreingruber** [comment]
It works, now also with the mob.sh tool....

That's IT. Combine 2 simple things and it's complicated :-D

File: .gitpod.dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM rust:latest
+
+# Install mob.sh tool
+RUN curl -sL install.mob.sh | sh
+
+# Install some rustup components that are available by default.
```
Comment (fredmorcos) [inline .gitpod.dockerfile:6]: Perhaps you meant "that are **not** available by default"?

**Review by fredmorcos — CHANGES_REQUESTED** [review]

File: .gitpod.dockerfile (hunk 1 above)
Comment (fredmorcos) [reply .gitpod.dockerfile:6]: I just realized it was me who added this >1 month ago... So here's one way, ladies and gentlemen, how a person could speak to their past self.

**Review by fredmorcos — CHANGES_REQUESTED** [review]

File: .gitpod.dockerfile (hunk 1 above)
Comment (haraldreingruber) [reply .gitpod.dockerfile:6]: Haha. yeah. These things happen with shared ownership :)

Comment (haraldreingruber) [reply .gitpod.dockerfile:6]: I guess we would need to look deeper, which ones are already provided by the default rust image... but let's see first if we want to continue this path?
Adding @HenningHolmDE into the loop.

**Comment by haraldreingruber** [comment]
Regarding the conflicts. Let's see if we still see value in this approach. Just updated the branch for testing and comparing...

**Comment by Mob-Programming-Meetup-Austria** [comment]
I guess we can close this PR now?

**Comment by haraldreingruber** [comment]
Yes. 👍

**Pull request closed without merging**